### PR TITLE
Update faiss build

### PR DESCRIPTION
**Pull request by ailzhang**
- faiss build with cuda80 and cuda91
- faiss-cpu not depend on cuda version
- remove cudatoolkit in faiss-gpu runtime.
nit: the missing newline was automatically removed by my editor... let me know if it's necessary. :P

### CODE REVIEW

File: conda/faiss-gpu/meta.yaml [hunk 1]
```diff
@@ -18,7 +18,6 @@ requirements:
     - python
     - numpy >=1.11
     - mkl
-    - cudatoolkit
```
Comment (soumith) [inline conda/faiss-gpu/meta.yaml]: why did you remove dependency on cudatoolkit?
are you assuming users has correct version of CUDA installed on system?

Comment (ailzhang) [reply conda/faiss-gpu/meta.yaml]: Mainly because we only have cudatoolkit8.0 in the channel. And if we want to install faiss-gpu with cuda90, it still installs cudatoolkit8.0 which is a bit weird.
